### PR TITLE
Section View

### PR DESCRIPTION
**Pull request by ismailsunni**
- CRUD for section
lesson_app

- Ordering for section (Sorry @myarjunar , it should be yours, I look at the wrong line in the excel)
section_ordering

- Unit test

cc @Gustry @myarjunar I have a migration in this PR.

### CODE REVIEW

File: django_project/lesson/views/section.py [hunk 1]
```diff
@@ -126,7 +132,329 @@ def get_queryset(self):
...
+        return context
+
+    def get_queryset(self):
+        """Get the queryset for this view.
+
+        :returns: Queryset for All Sections
+        :rtype: QuerySet
+        """
+
+        qs = Section.objects.all()
```
Comment (Gustry) [inline django_project/lesson/views/section.py]: This is already the default, no? Is the view not based on the model already?

File: django_project/lesson/views/section.py [hunk 2]
```diff
@@ -126,7 +132,329 @@ def get_queryset(self):
...
+        sections_json = request.body
+
+        try:
+            section_request = json.loads(sections_json)
+        except ValueError:
+            raise Http404(
+                'Error json values'
+            )
+
+        for sec in section_request:
```
Comment (timlinux) [inline django_project/lesson/views/section.py]: sec -> section

File: django_project/lesson/views/section.py (hunk 1 above)
Comment (ismailsunni) [reply django_project/lesson/views/section.py]: Ah yes. Thanks.

File: django_project/lesson/views/section.py (hunk 2 above)
Comment (ismailsunni) [reply django_project/lesson/views/section.py]: section is already used, I changed it with `for section_request in sections_request`

**Pull request merged**